README.md

```diff
@@ -39,22 +39,6 @@ The steps to build vary slightly depending on your distro.  The following set of
 
     This downloads a .NET SDK and a number of .NET packages needed to build .NET from source.
 
-    > On Linux distros other than Fedora 33, an additional bootstrapping step is required.  After running `prep.sh` above, run the following:
```

Comment:
I believe the bootstrap option is still necessary for building on arm64

Reply:
This issue would eliminate the need for the bootstrap option for arm64 - https://github.com/dotnet/source-build/issues/2758